tools/convert/main.cpp

```diff
@@ -72,6 +72,9 @@ static void PrintUsage(const char* exe_name)
     GFXRECON_WRITE_CONSOLE(
         "  --file-per-frame\tCreates a new file for every frame processed. Frame number is added as a suffix");
     GFXRECON_WRITE_CONSOLE("                  \tto the output file name.");
+    GFXRECON_WRITE_CONSOLE(
+        "  --frame-range\t\tFrame range to be converted. In order to retrieve Trim Trace State, frame 0 has "
+        "to be in frame range.");
```

Comment:
It would be good to tell the user the format for specifying the frame range here.

Reply:
I have updated the help message to:
Frame ranges to be converted. In order to retrieve Trim Trace State, frame 0 has to be in frame range.
Frame ranges should be specified in ascending order and cannot overlap. Example: 0-5,15,20,25-50,51.